lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -375,19 +378,22 @@ export class NativeInteractionClient extends BaseInteractionClient {
 
         // Get the preferred_cache domain for the given authority
         const authority = await this.getDiscoveredAuthority(request.authority);
-        const authorityPreferredCache = authority.getPreferredCache();
 
         // generate identifiers
         const idTokenObj = this.createIdTokenObj(response);
         const homeAccountIdentifier = this.createHomeAccountIdentifier(
             response,
             idTokenObj
         );
-        const accountEntity = this.createAccountEntity(
-            response,
+        const accountEntity = AccountEntity.createAccount(
             homeAccountIdentifier,
```

Comment:
It's out of scope of this PR, but something we can address later on. It would be great to be able to pass an object instead of plain params to make sure we can omit undefined ones and not accidentally mix string parameters.

Reply:
That's a great suggestion, let me see if I can squeeze that in here

Reply:
Made this change! Thanks for the suggestion!